### PR TITLE
DVX-37 Nginx, formatting & makefiles

### PR DESCRIPTION
**Pull request by jean-noumena**

### CODE REVIEW

File: webapp/src/RuntimeConfigurationProvider.tsx [hunk 1]
```diff
@@ -26,7 +26,10 @@ interface RuntimeConfigurationProviderProps {
  */
 export const loadRuntimeConfiguration =
     async (): Promise<RuntimeConfiguration> => {
-        const config_file = import.meta.env.VITE_ENV == 'LOCAL' ? '/config.json' : '/config-prod.json'
+        const config_file =
+            import.meta.env.VITE_ENV == 'LOCAL'
```
Comment (pavlo-noumena) [inline webapp/src/RuntimeConfigurationProvider.tsx:30]: do we need in `.env.properties` duplicated
` VITE_ENV == 'LOCAL' ` ?
I was expecting Prod or similar

File: .env [hunk 2]
```diff
@@ -1,3 +1,3 @@
 KEYCLOAK_ADMIN=admin
 KEYCLOAK_ADMIN_PASSWORD=password123
-PLATFORM_VERSION=2024.1.7
+PLATFORM_VERSION=2024.1.9
```
Comment (pavlo-noumena) [inline .env:3]: Same version should be used also in pom.xml, currently 
```
    <properties>
        <platform.version>2024.1.7</platform.version>
    </properties>
```

File: npl/pom.xml [hunk 3]
```diff
@@ -12,10 +12,6 @@
         <relativePath>../parent-pom</relativePath>
     </parent>
 
-    <properties>
-        <platform.version>2024.1.7</platform.version>
```
Comment (pavlo-noumena) [inline npl/pom.xml:16]: `platform.version` used for npl-maven-plugin versioning:
image, 
npl-maven-plugin should be set alternatively if removed, currently causing [failed build](https://github.com/NoumenaDigital/npl-integrations/actions/runs/10266361399/job/28404537668?pr=15)
image

File: paas.mk [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+download-cli: export API_URL=https://api.github.com/repos/NoumenaDigital/npl-cli/releases/tags/$(RELEASE_TAG)
+download-cli:
+	curl -s -H "Authorization: token $(GITHUB_USER_PASS)" $(API_URL) \
+		| jq --arg CLI_OS_ARCH "$(CLI_OS_ARCH)" '.assets[] | select(.name == $$CLI_OS_ARCH) | .url' -r \
+		| xargs -t -n 2 -P 3 curl -sG -H "Authorization: token $(GITHUB_USER_PASS)" -H "Accept: application/octet-stream" -Lo cli
+	chmod +x cli
+
+.PHONY: create-app
+create-app:
+	./cli app create -org $(NC_ORG) -engine 2024.1.3 -name $(NC_APP_NAME) -provider MicrosoftAzure -trusted_issuers '["https://keycloak-$(NC_ORG_NAME)-$(NC_APP_NAME).$(NC_DOMAIN)/realms/$(NC_APP_NAME)"]'
```
Comment (pavlo-noumena) [inline paas.mk:66]: Maybe extract to PAAS  property to have clear why engine version is different from regular platform?

File: webapp/src/components/CreateIouDialog.tsx [hunk 5]
```diff
@@ -118,9 +116,7 @@ export const CreateIouDialog: React.FC<{
                             variant="outlined"
                             value={payee}
                             type={'email'}
-                            onChange={(e) =>
```
Comment (pavlo-noumena) [inline webapp/src/components/CreateIouDialog.tsx:121]: Can we use Sonar or at least github actions for linters

Comment (jean-noumena) [reply webapp/src/components/CreateIouDialog.tsx:121]: Yes, please :D 
https://noumenadigital.atlassian.net/browse/ST-4191

**Review by pavlo-noumena — APPROVED** [review]

**Pull request merged**